src/components/EpicView/helpers/queryFetchers.ts

```diff
@@ -0,0 +1,6 @@
+import {Issue} from "../../../../types";
+
+export const getEpics = (
+    projectKey: string | undefined
+): Promise<Issue[]> =>
+    window.provider.getEpicsByProject(projectKey || "")
```

Comment:
This might need an additional last line (good practice to avoid issues with some tools)